cmd/buildah/manifest.go

```diff
@@ -0,0 +1,653 @@
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type manifestCreateOpts = struct {
+	osOverride, archOverride string
+	all                      bool
+}
+type manifestAddOpts = struct {
+	osOverride, archOverride          string
+	os, arch, variant, osVersion      string
+	features, osFeatures, annotations []string
+	all                               bool
+}
+type manifestRemoveOpts = struct{}
+type manifestAnnotateOpts = struct {
+	os, arch, variant, osVersion      string
+	features, osFeatures, annotations []string
+}
+type manifestInspectOpts = struct{}
+type manifestPushOpts = struct {
+	purge, quiet, all, tlsVerify                          bool
+	authfile, certDir, creds, digestfile, signaturePolicy string
+}
+
+func init() {
+	var (
+		manifestDescription         = "\n  Creates, modifies, and pushes manifest lists and image indexes."
+		manifestCreateDescription   = "\n  Creates manifest lists and image indexes."
+		manifestAddDescription      = "\n  Adds an image to a manifest list or image index."
+		manifestRemoveDescription   = "\n  Removes an image from a manifest list or image index."
+		manifestAnnotateDescription = "\n  Adds or updates information about an entry in a manifest list or image index."
+		manifestInspectDescription  = "\n  Display the contents of a manifest list or image index."
+		manifestPushDescription     = "\n  Pushes manifest lists and image indexes to registries."
+		manifestCreateOpts          manifestCreateOpts
+		manifestAddOpts             manifestAddOpts
+		manifestRemoveOpts          manifestRemoveOpts
+		manifestAnnotateOpts        manifestAnnotateOpts
+		manifestInspectOpts         manifestInspectOpts
+		manifestPushOpts            manifestPushOpts
+	)
+	manifestCommand := &cobra.Command{
+		Use:   "manifest",
+		Short: "Manipulate manifest lists and image indexes",
+		Long:  manifestDescription,
+		Example: `buildah manifest create localhost/list
+  buildah manifest add localhost/list localhost/image
+  buildah manifest annotate --annotation A=B localhost/list localhost/image
+  buildah manifest annotate --annotation A=B localhost/list sha256:entryManifestDigest
+  buildah manifest remove localhost/list sha256:entryManifestDigest
+  buildah manifest inspect localhost/list
+  buildah manifest push localhost/list transport:destination`,
+	}
+	manifestCommand.SetUsageTemplate(UsageTemplate())
+	rootCmd.AddCommand(manifestCommand)
+
+	manifestCreateCommand := &cobra.Command{
+		Use:   "create",
+		Short: "Create manifest list or image index",
+		Long:  manifestCreateDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return manifestCreateCmd(cmd, args, manifestCreateOpts)
+		},
+		Example: `buildah manifest create mylist:v1.11`,
+	}
+	manifestCreateCommand.SetUsageTemplate(UsageTemplate())
+	flags := manifestCreateCommand.Flags()
+	flags.BoolVar(&manifestCreateOpts.all, "all", false, "add all of the lists' images if the images to add are lists")
```

Comment:
You're talking about "add" here but the action is create?

Reply:
Whoops, the `create` command can optionally be told to add one or more images to the list at the same time, and this is a reference to that.  Adding examples that take advantage of that to the help output.